source/scenes/gameplay/PPGameScene.cpp

```diff
@@ -184,8 +186,6 @@ void GameScene::update(float timestep) {
     // Check if the level is complete
     if (activeCanvases.empty() && !_congratulations) {
         //Gradually clear out the splatters
-         _splash->update(timestep,
-                    Color4::CLEAR, Vec2::ZERO);
```

Comment:
Why delete this?

Reply:
Oh it's not necessary. I think I was testing this when trying to debug shader on this branch and just left it in because it was an excess update call anyways.

Reply:
I can also undo this and remove this in the hotfix